extensions/dacpac/src/test/index.ts

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 const path = require('path');
-import * as testRunner from 'vscodetestcover';
+const testRunner = require('vscode/lib/testrunner');
 
```

Comment:
Why use require here?

Reply:
It's what all the other tests are using.